backend/api/chatLink/utils/validateChannel.test.ts

```diff
@@ -0,0 +1,63 @@
+import channelValid, { CHANNEL_STATE } from './validateChannel';
+import db from "../../../db";
+
+jest.mock('../../../db');
+
+describe('channelValid', () => {
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    // test if channel is provided otherwise throw new Error("channel - required param");
+    it('should throw an error is the channel is not provided', async () => {
+        await expect(channelValid('')).rejects.toThrow("channel - required param");
+    });
+
+    it('should return NOT_FOUND if the channel does not exist in the db', async () => {
+        const findOneFromDBMock = db.findOneFromDB as jest.MockedFunction<typeof db.findOneFromDB>;
+        findOneFromDBMock.mockResolvedValueOnce(null);
```

Comment:
`db.findOneFromDB = jest.fn().mockResolvedValueOnce(null);`
this should work too.

Reply:
Thank you, I'll make the change.